src/sentry/utils/canonical.py

```diff
@@ -94,7 +94,7 @@ def __init(self, data):
             legacy = settings.PREFER_CANONICAL_LEGACY_KEYS
         norm_func = legacy and get_legacy_name or get_canonical_name
         self._norm_func = norm_func
-        self.data = collections.OrderedDict()
+        self.data = {}
```

Comment:
Is this intentional?

Reply:
That was just me testing something, it's not part of the final PR diff. You probably didn't refresh your browser.